documentation/modules/configuring/con-partition-reassignment.adoc

```diff
@@ -7,6 +7,20 @@
 = Partition reassignment tool overview
 
 [role="_abstract"]
+The partition reassignment tool provides the following capabilities for managing Kafka partitions and brokers:
+
+Scaling clusters up and down:: Scale your cluster up and down by adding or removing brokers, and move Kafka partitions from heavily loaded brokers to under-utilized brokers. 
```

Comment:
Scaling the cluster is not a capability of the reassignment tool. It only has the ability to redistribute the partition replicas after/before the cluster is scaled.

Reply:
+1